src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloadsFixer.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Currently working on a fixer. Will probably add it in a separate PR.

Reply:
If fixer is going to be a separate PR, please remember to remove this file from this PR before merge. Thanks!

Reply:
The unit tests have a using directive that is calling this. Can I create the VerfyCS alias without a fixer?

Reply:
Change the fixer to `Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider` :smile:

Reply:
Thanks. Good to know that exists.